README.md

```diff
@@ -34,21 +34,28 @@ You can instantiate the object like this:
 
 ```java
 // add Blockbird Audit
-bbAudit = BlockbirdAudit.getInstance("URL-of-API","dbKey","dbSecret");
+bbAudit = new BlockbirdAudit(boolean sandbox, String apiUrl);
+OR
+bbAudit = new BlockbirdAudit(boolean sandbox);
+OR
+bbAudit = new BlockbirdAudit(String apiUrl);
+OR
+bbAudit = new BlockbirdAudit();
 ```
-On instantion, the Tables and Columns that have been flagged as containing Personal Data during the On-boarding phase will be retrieved. 
 
-Then you add queries to your packet:
+Then initialize the app:
 
 ```java
-bbAudit.addQuery(clientUserId, clientRole, clientTable, clientRole, action("Create" | "Read" | "Update" | "Delete"), actionDate, rowsAffected);
-
+bbAudit.initialize(String dbId, String dbSecret);
```

Comment:
Why `dbId`? I updated all `dbId`'s to`dbKey`.

Reply:
@micklynch sry I don't know if I tell you, but we are now using `dbKey` instead of `dbId` (`dbId` only to reference internal use of the database id). For an external developer it is most common to use `dbKey`